### ISSUE TITLE
Mouse not working for me too

### ISSUE THREAD

**Issue by jomidatu** [fqlx/XboxKeyboardMouse#178]
Hi,
Is there a workaround to get the mouse movement work? I can use the Keyboard and the mouse button but not move. Tried reinstall and different Xbox-apps. The settings bar at the top of the screen is visible when i move the Mouse in fullscreen. 

Would be great to use this program.. Thank you!

**Comment by TheNumba1** [comment]
I'm not exactly sure if its what fixed mine, but I did a few things so try them all: I change my mouse tick rate to 9, I changed both sensitivities to 14200, I use dead zoning default with a 500 final modifier, and what I think TRULY FIXED IT was: go into the xboxkeyboardmouse master file, click downloads, profiles, choose your custom, not the default, and under mouse settings, change the deadzone to 6000

**Comment by UIPL3B** [comment]
Hey @TheNumba1, where exactly can you disable the curser when streaming? It is still coming up and click on things. Mouse still not working for me

**Comment by UIPL3B** [comment]
Hey @TheNumba1, are you opening the XKB application via the "Installer" file or "Downloads" file?

**Comment by TheNumba1** [comment]
Via the downloads file, profiles will be inside, click your custom profile
and change deadline under the mouse settings,         and to change the
cursor, just run the application, click settings, go to the settings tab
and uncheck show cursor

**Comment by gruberdominik1909** [comment]
mous dont work how app you use

**Comment by UIPL3B** [comment]
> Via the downloads file, profiles will be inside, click your custom profile
> and change deadline under the mouse settings,         and to change the
> cursor, just run the application, click settings, go to the settings tab
> and uncheck show cursor
> …

Thanks for that. I have actually tried all that but for some odd reason, the Downloads folder version doesn't detect the Xbox app at all... I'll keep trying but thanks for the help :)

**Comment by TheNumba1** [comment]
What do you mean doesnt detect the xbox app? Did you ever unzip the main
folder into it's own folder

**Comment by gruberdominik1909** [comment]
picture we need step by step are 100 people  hear with same problem mous not work thanks

**Comment by UIPL3B** [comment]
> What do you mean doesnt detect the xbox app? Did you ever unzip the main
> folder into it's own folder
> …
Yes, it has been unzipped... When XKM sees that the Xbox app is the actively used application on your computer it turns green, when turned on. When I turn it on and click over to the Xbox app it stays blue. 

Red = XKM not active / Off
Blue = XKM active / on but not detecting Xbox app or Xbox app not being used
Green = XKM active and dectecting the Xbox app

**Comment by TheNumba1** [comment]
Ya I realize that but u said in the downloads folder it didnt connect
